config/karma.saucelabs.js

```diff
@@ -50,7 +50,18 @@ const packageConfigs = {
     // Messaging currently only supports these browsers.
     browsers: ['Chrome_Windows', 'Firefox_Windows', 'Edge_Windows']
   },
-  firestore: {
+  // Firestore unit tests have OOM problems compiling with Babel for IE.
+  // Firestore integration/firestore tests do run on all browsers.
+  'packages/firestore': {
```

Comment:
The key is a bit confusing until I read more code below. Can we use the package name (`@firebase/firestore`) instead to make it easier to understand?

Reply:
Done.